### PR TITLE
Add Windows containerd job to release-informing dashboard

### PR DESCRIPTION
**Pull request by jsturtevant**
Containerd went stable for Windows in 1.20, This adds the Windows containerd job to Release Informing dashboard: https://testgrid.k8s.io/sig-release-master-informing#Summary

It also bumps the job to run every 3 hours in to meet the requirements for being considered for Release-blocking: https://github.com/kubernetes/sig-release/blob/master/release-blocking-jobs.md#release-blocking-criteria-and-dashboard

/sig windows
/sig release

cc: @ravisantoshgudimetla @marosset @jayunit100 @chewong

### CODE REVIEW

**Review by chewong — APPROVED** [review]
/lgtm
/hold
for others to review

**Comment by dims** [comment]
/hold cancel

thanks @jsturtevant !

**Comment by justaugustus** [comment]
For posterity, post-merge lgtm+approve for SIG Release:
/lgtm
/approve

Let's make sure we're getting approval from SIGs **_before_** adding jobs to their boards.
ref: https://kubernetes.slack.com/archives/C2C40FMNF/p1611148357059000

FYI @kubernetes/release-engineering @kubernetes/release-team-leads

**Review by justaugustus — CHANGES_REQUESTED** [review]
@jsturtevant @chewong -- As a follow-up, I'd suggest adding the fork annotation.

https://github.com/kubernetes/test-infra/blob/7d498063b6401909edf2158bed0b03fe2909cb50/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L268-L269

That'll make sure this job doesn't get missed when we create the branch jobs later in the 1.21 release cycle.

**Comment by jsturtevant** [comment]
@justaugustus Thanks for the heads up on the annotation. Sorry about not informing you via slack... Will make sure to do that in the future.

**Pull request merged**
